src/main/scala/org/broadinstitute/dsde/firecloud/service/TrialServiceSupport.scala

```diff
@@ -34,12 +36,13 @@ trait TrialServiceSupport extends LazyLogging {
   def makeSpreadsheetValues(managerInfo: UserInfo, trialDAO: TrialDAO, thurloeDAO: ThurloeDAO, majorDimension: String, range: String)
     (implicit executionContext: ExecutionContext): Future[ValueRange] = {
     val projects: Seq[TrialProject] = trialDAO.projectReport
-    val trialStatuses = Future.sequence(projects.map { p =>
-      if (p.user.isDefined)
-        thurloeDAO.getTrialStatus(p.user.get.userSubjectId, managerInfo)
-      else
-        Future[Option[UserTrialStatus]](None)
-    })
+    val trialStatuses = Future.sequence(
+      projects.foldLeft(Seq[Future[UserTrialStatus]]())((result, p) => {
+        if (p.user.isDefined)
+          result ++ Seq(thurloeDAO.getTrialStatus(p.user.get.userSubjectId, managerInfo))
+        else
+          result
+      }))
```

Comment:
mostly style, but I generally find `fold`s to be hard to read and avoid unless they're really needed. Your call; would either of these be better?
```
    val trialStatuses = Future.sequence(
      projects.filter(p => p.user.isDefined)
        .map(p => thurloeDAO.getTrialStatus(p.user.get.userSubjectId, managerInfo)))
```
or
```
    val trialStatuses = Future.sequence(
      projects.map {
        case p if p.user.isDefined => thurloeDAO.getTrialStatus(p.user.get.userSubjectId, managerInfo))
      })
```
(again, untested)